server/issue.go

```diff
@@ -1094,3 +1095,32 @@ func (p *Plugin) GetIssueByKey(instanceID, mattermostUserID types.ID, issueKey s
 	}
 	return issue, nil
 }
+
+func (p *Plugin) GetIssueDataWithAPIToken(issueID string, instanceID string) (*jira.Issue, error) {
+	client := &http.Client{}
+	req, err := http.NewRequest("GET", fmt.Sprintf("%s/rest/api/2/issue/%s", instanceID, issueID), nil)
+	if err != nil {
+		return nil, err
```

Comment:
why are we not logging errors here?

Reply:
check everywhere

Reply:
The errors are getting logged in a parent function. Added wrapper for clarity.